ee/tabby-db/src/invitations.rs

```diff
@@ -22,14 +23,7 @@ impl DbConn {
         skip_id: Option<i32>,
         backwards: bool,
     ) -> Result<Vec<InvitationDAO>> {
-        let query = Self::make_pagination_query(
-            "invitations",
-            &["id", "email", "code", "created_at"],
-            limit,
-            skip_id,
-            backwards,
-        );
-
+        let query = pagination_query!(InvitationDAO FROM "invitations" ["id", "email", "code", "created_at"!], limit, skip_id, backwards);
```

Comment:
```suggestion
        let query = query_pagination_as!(InvitationDAO, "invitations", ["id", "email", "code", "created_at"!], limit, skip_id, backwards);
```

Can the usage being updated to this style?

Reply:
Can do